draft-packages/button/KaizenDraft/Button/styles.scss

```diff
@@ -349,6 +349,31 @@ $caButton-verticalPaddingForm: calc(
       color: $color-white;
     }
   }
+
+  &%caButtonDestructive {
+    &:not(:disabled) {
+      background: $color-red-300;
+      border-color: $border-borderless-border-color;
+      color: $color-purple-800;
```

Comment:
I know that this is probably the case in the other ones that this was coped from, but these three properties are repeated again below for `&:disabled, &.working`. I think this can DRY'd up and set once at the `&%caButtonDestructive` level?

Reply:
Good callout, thanks @dougmacknz. 

I gave this a shot, for Reversed Destructive, and for the other variants (Reversed Secondary, Primary etc).

However, this unfortunately broke the button styles in all cases. It look's like the extra CSS selector specificity is currently needed in each of the sub selectors for the desired styles to take precedence. The current repetition might be required for the moment until there's a more general refactor.

Might make a good follow up ticket?

Reply:
Ah right, yeah ok don't worry about it. This file is in a bit of a need of a refactor I think